### DISCUSSION TITLE
About get_connection, get_variable, and get_config.

### DISCUSSION THREAD

**Discussion by hnguye99** [apache/airflow#25247] [Q&A]
https://github.com/apache/airflow/blob/b815e221e828aa039e166378c3e393d144b174e8/airflow/providers/hashicorp/secrets/vault.py#L194

What is the intended difference between get_connection, get_variable and get_config? How are they 'supposed' to work? Is this documented anywhere?

**Comment by potiuk** [comment]
Did you search for those terms in the docs ? What did you find ?
